mllib/src/main/scala/org/apache/spark/ml/classification/RandomForestClassifier.scala

```diff
@@ -93,9 +93,12 @@ final class RandomForestClassifier(override val uid: String)
     val oldDataset: RDD[LabeledPoint] = extractLabeledPoints(dataset)
     val strategy =
       super.getOldStrategy(categoricalFeatures, numClasses, OldAlgo.Classification, getOldImpurity)
-    val oldModel = OldRandomForest.trainClassifier(
-      oldDataset, strategy, getNumTrees, getFeatureSubsetStrategy, getSeed.toInt)
-    RandomForestClassificationModel.fromOld(oldModel, this, categoricalFeatures)
+    // val oldModel = OldRandomForest.trainClassifier(
+    //   oldDataset, strategy, getNumTrees, getFeatureSubsetStrategy, getSeed.toInt)
+    // RandomForestClassificationModel.fromOld(oldModel, this, categoricalFeatures)
+    val trees = RandomForest.run(oldDataset, strategy, getNumTrees, getFeatureSubsetStrategy,
+      getSeed).map(_.asInstanceOf[DecisionTreeClassificationModel])
```

Comment:
move `.map` to a new line, e.g.:

~~~scala
val trees = RandomForest.run(
    oldDataset, strategy, getNumTrees, getFeatureSubsetStrategy, getSeed)
  .map(_.asInstanceOf[DecisionTreeClassificationModel])
~~~